.github/workflows/main.yml

```diff
@@ -25,6 +25,9 @@ jobs:
       - name: 'Checkout'
         uses: actions/checkout@v2 # required!
 
+      - name: 'Update version string'
+        run: yq '.version' pubspec.yaml > lib/assets/version.txt
```

Comment:
Are we sure that `yq` is installed in the container that runs the build?

Reply:
Yeah. The `ubuntu-latest` Github runner has `yq` installed: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md